File: consorcio.java

//Esse Algoritmo mostra a quantidade de parcelas, as parcelas que ja foram pagas e tambem o valor e o saldo devedor.

package JAVA;
import javax.swing.JOptionPane; // biblioteca para as caixa de saída interação;
class consorcio {
        public static void main(String args[]) { // correponder ao metodo principal
         
        float QtdParcelas, Qtdpg, ValorAtual, Valorpg, SaldoP;
        QtdParcelas = Float.parseFloat(JOptionPane.showInputDialog("quantas parcelas tem o consorcio: "));
        Qtdpg = Float.parseFloat(JOptionPane.showInputDialog("Quantas parcelas ja foram pagas: "));         //Definição das entradas;
        ValorAtual = Float.parseFloat(JOptionPane.showInputDialog("qual o valor da parcela: "));

        Valorpg = ValorAtual * Qtdpg;
        SaldoP =  ValorAtual * (Qtdpg - QtdParcelas); // processamento dos dados;
        
        JOptionPane.showMessageDialog(null, "A quantidade total de parcelas do consorcio....:"+QtdParcelas);
        JOptionPane.showMessageDialog(null,"As parcelas pagas sao..........................:"+Qtdpg);            //saidas com apresentação das caixas; 
        JOptionPane.showMessageDialog(null,"O valor atual do consorcio......................:"+ValorAtual);
        JOptionPane.showMessageDialog(null,"O valor do que já foi pago .....................:"+Valorpg);
        JOptionPane.showMessageDialog(null,"O saldo devedor e ..............................:"+SaldoP);

        JOptionPane.showMessageDialog(null,"A quandidade de parcelas e = "+QtdParcelas+ 
                                                           "\n já foram pagas e = "+Qtdpg+                // Conjunto das saídas em uma só caixa de texto;
                                                           "\n O valor atual é de = "+ValorAtual+
                                                           "\n O valor que foi pago = "+Valorpg+
                                                           "\n O saldo devedor = "+SaldoP);
                                            
                                                        
    

    


    

         
        }    
}

File: consorcio.c
// O algoritmo  mostra quantas parcelas tem no consorcio, quantidade de parcelas, o valor que foi pago e
// seu saldo devedor;

#include <stdio.h> 
int main () //metodo principal 
{
    float QtdParcelas, QtdPPg , ValorAtual, ValorPg, SaldoP; // quantas variáveis 
    
    printf("\n A quantidade de parcelas do Consorcio: "); // entradas; 
    scanf("%f",&QtdParcelas);

    printf("\n A quantidades de parcelas que foram pagas: "); // entradas;
    scanf("%f",&QtdPPg);

    printf("\n Valor atual das parcelas e R$ ");
    scanf("%f",&ValorAtual);

    ValorPg = ValorAtual * QtdPPg; // Processamento; 

    SaldoP = ValorAtual * (QtdParcelas - QtdPPg); //processamento;

    //printf("\n A quantidade de parcelas do seu consorcio............:%f",QtdParcelas);
    //printf("\n A quantidade de parcelas que foram pagas.............:%f",QtdPPg);
    //printf("\n Atual o senhor paga .................................:R$ %f",ValorAtual);  // Mostra separado como  valor 
    //printf("\n O valor que foi pago pelo pago........................:R$ %f",ValorPg);
    //printf("\n O valor pentende.......................................:R$ %f",SaldoP);

    printf("\n Senhor(a)");
    printf("\n A quantidade de parcelas sao = %f, foram pagas = %f parcelas. \n O  valor atual da parcela R$ %f , foi pago R$ %f  \n e seu saldo Devededor R$ %f",QtdParcelas,QtdPPg,ValorAtual,ValorPg,SaldoP); // Conjunto de todos a informações necessarias;

 printf("\n\n>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> O esperamos ter ajudado!<<<<<<<<<<<<<<<<<<<<");
 return 0; //mostra na tela;



}